.circleci/src/workflows/delivery_production_native.yml

```diff
@@ -42,18 +40,34 @@ jobs:
       production_delivery: true
       requires:
         - build_android_nuernberg
+#  - build_android:
+#      name: build_android_koblenz
+#      buildConfig: "koblenz"
+#      requires:
+#        - bump_version
+#        - check_frontend
+#      context:
+#        - credentials-repo
+#        - credentials-ehrenamtskarte
+#  - deliver_android:
+#      name: deliver_android_koblenz
+#      buildConfig: "koblenz"
+#      context:
+#        - tuerantuer-google-play
+#      production_delivery: true
+#      requires:
+#        - build_android_koblenz
```

Comment:
:question: I assume this will be uncommented in a later issue?

Reply:
yes we want to block production release until it is ready. #1427